app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -155,7 +182,9 @@ def presenter
           _, document_list = search_results(params)
           curation_concern = document_list.first
           raise CanCan::AccessDenied unless curation_concern
-          show_presenter.new(curation_concern, current_ability, request)
+          pres = show_presenter.new(curation_concern, current_ability, request)
+          raise WorkflowAuthorizationException if pres.parent_presenter.nil?
```

Comment:
`raise WorkflowAuthorizationException if pres.parent_presenter.blank?` might be more robust?

Reply:
Done.